metaflow/plugins/aws/step_functions/step_functions_client.py

```diff
@@ -75,6 +83,24 @@ def terminate_execution(self, state_machine_arn, execution_arn):
         #TODO
         pass
 
+    def _default_logging_configuration(self, log_execution_history):
+        if log_execution_history:
+            return {
+                'level': 'ALL',
+                'includeExecutionData': True,
+                'destinations': [
+                    {
+                        'cloudWatchLogsLogGroup': {
+                        'logGroupArn': SFN_EXECUTION_LOG_GROUP_ARN
```

Comment:
@savingoyal 
```suggestion
                        'logGroupArn': f'{SFN_EXECUTION_LOG_GROUP_ARN}:*'
```
It seems a little strange to have to specify the `:*` in the config, versus just the arn of the log group

Reply:
Isn't the ARN for the log group actually `arn:aws:logs:us-west-2:<redacted>:log-group:/aws/foo:*`?

Reply:
I am looking at the `return values` section here - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html

Reply:
ah.. you are right - never mind then - I was looking at another page which made me think otherwise